### ISSUE TITLE
give exotic matter negative mass

### ISSUE THREAD

**Issue by L5108313** [Eggs-D-Studios/wos-issues#434]
### Guidelines

- [X] I checked for duplicate suggestions
- [X] My suggestion intends to add or improve something

### Version

Main (Production)

### Topic

A game mechanic (e.g. a part, a feature, etc)

### Why are you making your suggestion?

Currently, the exotic matter part is useless by itself, and only makes rtgs. This is very boring. Exotic matter having negative mass also makes sense, because that is the definition of what exotic matter is.

### What do you propose? What should change, or what should be added?

Somehow make the exotic matter part act as if it had negative mass. Im not sure how possible this is, but i think it could potentially be cool. Also give exotic matter like 200 malleability so its actually like usuable

**Comment by Hexcede** [comment]
What would this actually mean

**Comment by Apethesis** [comment]
> What would this actually mean

Exotic matter could have negative mass/density and malleability so it could be potentially used for lightening ships in the endgame. This would be an easy improvement over them just being a crafting resource

**Comment by Hexcede** [comment]
> > What would this actually mean
> 
> Exotic matter could have negative mass/density and malleability so it could be potentially used for lightening ships in the endgame. This would be an easy improvement over them just being a crafting resource

What would negative mass actually mean though

**Comment by L5108313** [comment]
i think negative mass could just mean that it subtracts mass from the assembly its part of, so it just makes stuff lighter. mass doesnt really play much of a role in wos right now though.

**Comment by hail12pink** [comment]
> mass doesnt really play much of a role in wos right now though.

yeah. problem with balancing with mass in wos: who freaking cares it's space there's either no or very little gravity more than half the time anyways
